forloop_modules/function_handlers/input_waiter.py

```diff
@@ -0,0 +1,26 @@
+# input_waiter.py
+from forloop_modules.globals.variable_handler import LocalVariable, variable_handler
+from forloop_modules.queries.node_context_requests_backend import post_console_log
+import forloop_modules.flog as flog
+
+_pending: dict[str, str] = {}
+_prompts: dict[str, str] = {} 
+
+def register_pending_input(node_uid: str, output_variable: str, prompt_text: str = None):
+    _pending[node_uid] = output_variable
+    if prompt_text:
```

Comment:
I like being more explicit with strings - as this looks like "boolean" evaluation and we don't know if the variable is "string" -> if prompt_text is not None:

Reply:
But maybe thats just matter of prefereence